jdbc/core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/statement/ReadwriteSplittingPreparedStatementTest.java

```diff
@@ -118,4 +118,65 @@ public void assertGetGeneratedKeysWithPrimaryKeyIsNullInTransactional() throws S
             connection.commit();
         }
     }
+    
+    @Test
+    public void assertGetAutoGeneratedKeysAfterExecuteBatch() throws SQLException {
```

Comment:
Can you move these case to e2e test case? We need to test different scenarios such as sharding, read-write splitting, etc.

Reply:
You can add test case in `cases/dml/dml-integration-insert.xml` file, and then execute `BatchDMLE2EIT`.

Reply:
Alright,I will try this week